android/capacitor/src/main/java/com/getcapacitor/plugin/Toast.java

```diff
@@ -23,7 +30,22 @@ public void show(PluginCall call) {
       duration = android.widget.Toast.LENGTH_LONG;
     }
 
+
     android.widget.Toast toast = android.widget.Toast.makeText(getContext(), text, duration);
+
+    String position = call.getString("position", "bottom");
+
+    if("top".equals(position)) {
+      toast.setGravity(GRAVITY_TOP, 0, BASE_TOP_BOTTOM_OFFSET);
+    } else  if("bottom".equals(position)) {
+      toast.setGravity(GRAVITY_BOTTOM, 0, BASE_TOP_BOTTOM_OFFSET);
```

Comment:
This also changes the behaviour, so I wouldn't do anything if set to bottom, to keep the old behaviour.

Reply:
Is it not weird to have a margin at the top but no at bottom? IIRC on android the toast sticks to the bottom, I found it not as pretty as with a little margin.

Reply:
I don't know if it's weird or not, but this PR changes the previous behaviour, moving the default toast position.

Reply:
Ok I changed it, and put also 40 as top offset.